vnext/Microsoft.ReactNative/JsiApi.cpp

```diff
@@ -578,8 +597,20 @@ IJsiHostObject JsiRuntime::GetHostObject(JsiObjectRef obj) try {
 
 JsiHostFunction JsiRuntime::GetHostFunction(JsiObjectRef func) try {
   auto funcPtr = AsPointerValue(func);
-  // auto hostFunction = m_runtime->getHostFunction(AsFunction(func));
-  // TODO: implement mapping
+  auto const &jsiFunc = AsFunction(&funcPtr);
+  auto &rt = *m_runtime;
+  int64_t hostFunctionId = static_cast<int64_t>(rt.global()
+                                                    .getPropertyAsObject(rt, "__jsi_pal")
+                                                    .getPropertyAsFunction(rt, "getHostFunctionId")
+                                                    .call(rt, jsiFunc)
+                                                    .getNumber());
```

Comment:
Should we need to check for undefined here? And assert with detailed message that we couldn't find the method?

Reply:
If something is missing, then we get an exception will all details.

---
In reply to: [537922680](https://github.com/microsoft/react-native-windows/pull/6693#discussion_r537922680) [](ancestors = 537922680)